erpnext/setup/setup_wizard/operations/taxes_setup.py

```
@@ -147,8 +167,21 @@ def make_item_tax_template(company_name, template):
 			if 'tax_rate' not in tax_row:
 				tax_row['tax_rate'] = account_data.get('tax_rate')
 
-	return frappe.get_doc(template).insert(ignore_permissions=True)
-
+	doc = frappe.get_doc(template)
+	doc.flags.ignore_links = True
+	doc.flags.ignore_validate = True
+	doc.insert(ignore_permissions=True)
```

Is all this ignoring really necessary? I don't remember getting any errors, at least for the german setup, and if there are any they might be worth fixing...

---

This is not to ignore any errors but to reduce the setup time. India has a lot of fixtures that are created during setup which takes a lot of time. Have not done any profiting yet so cannot say if this helps or not. Will remove if there is no significant change

---

I see. Could you please add a comment mentioning this?